app/code/community/Oyst/OneClick/Model/Observer.php

```diff
@@ -128,4 +128,59 @@ public function appendOneClickButton(Varien_Event_Observer $observer)
             }
         }
     }
+
+    /**
+     * Tag conversion.
+     *
+     * @param Varien_Event_Observer $observer
+     */
+    public function tagConversion(Varien_Event_Observer $observer)
+    {
+        $order = $observer->getOrder();
+        $url = Mage::app()->getStore($order->getStoreId())->getBaseUrl(Mage_Core_Model_Store::URL_TYPE_LINK);
+
+        if ($order->getOystOrderId()) {
+            $url .= 'oyst_oneclick/notifications/index';
+        } else {
+            $url .= 'checkout/onepage/success/';
+        }
+
+        $cookie = str_replace('\n', '', $this->execute('https://api.oyst.com/session'));
+
+        $json_array = array(
+            "referer" => $url,
```

Comment:
change for single quote and use double quote only when you have to process what is inside the double quote